scripts/nemo/qnn/parakeet-tdt-ctc/transducer/run.sh

```
@@ -0,0 +1,28 @@
+num_frames=1000
+if [ $# -ge 1 ]; then
+  num_frames="$1"
+fi
+
+
+model_id=nvidia/parakeet-tdt_ctc-110m
+if [ $# -ge 2 ]; then
+  model_id="$2"
+fi
+
+python3 ./wrapper.py --max-len $num_frames --model-id $model_id
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)